lib/unsupported-features/node-globals.js

```diff
@@ -135,14 +135,21 @@ const nodeGlobals = {
     // module.crypto
     crypto: {
         ...crypto.webcrypto,
-        [READ]: { experimental: ["17.6.0", "16.15.0"] },
+        [READ]: { experimental: ["17.6.0", "16.15.0"], supported: ["23.0.0"] },
+    },
+    Crypto: {
+        [READ]: { experimental: ["17.6.0", "16.15.0"], supported: ["23.0.0"] },
+    },
+    CryptoKey: {
+        [READ]: { experimental: ["17.6.0", "16.15.0"], supported: ["23.0.0"] },
     },
-    Crypto: { [READ]: { experimental: ["17.6.0", "16.15.0"] } },
-    CryptoKey: { [READ]: { experimental: ["17.6.0", "16.15.0"] } },
     SubtleCrypto: { [READ]: { experimental: ["17.6.0", "16.15.0"] } },
 
     // module.events
-    CustomEvent: events.CustomEvent,
+    CloseEvent: { [READ]: { supported: ["23.0.0"] } },
+    CustomEvent: {
+        [READ]: { experimental: ["18.7.0", "16.17.0"], supported: ["23.0.0"] },
```

Comment:
Okay, so checking on all of these, you're correct, it should be:

- `20.13.0` - [CHANGELOG_V20.md](https://github.com/nodejs/node/blob/main/doc/changelogs/CHANGELOG_V20.md#eventsdoc-mark-customevent-as-stable)
- `22.1.0` - [CHANGELOG_V22.md](https://github.com/nodejs/node/blob/main/doc/changelogs/CHANGELOG_V22.md#22.1.0:~:text=Atlow)%20%2352515-,%5B3026401be1%5D%20%2D%20events%2Cdoc%3A%20mark%20CustomEvent%20as%20stable%20(Daeyeon%20Jeong)%20%2352618,-%5Bc6e0fe2f22%5D%20%2D)

It should not include `23.0.0`.

```suggestion
        [READ]: { experimental: ["18.7.0", "16.17.0"], supported: ["20.13.0", "22.1.0"] },
```

Reply:
@scagood This is the global one, which is only added as of v23

https://nodejs.org/docs/latest-v23.x/api/globals.html#customevent
![](https://github.com/user-attachments/assets/91c2d377-1522-4d05-8026-90c824d8b475)

The one in `events` module was already correct

https://nodejs.org/docs/latest-v23.x/api/events.html#class-customevent
![](https://github.com/user-attachments/assets/6fb5fba1-ab69-45bc-9018-2404ec17cfd9)

Reply:
Gotcha